src/harness/BenchmarkDotNet.Extensions/BenchmarkDotNet.Extensions.csproj

```
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

is this change necessary? 
```suggestion
    <TargetFramework>netstandard2.0</TargetFramework>
```

---

This was from something else I was testing that leaked into the commit. I removed the net5.0 framework, but not the Frameworks tag. I'll fix this.